scripts/ci

```diff
@@ -12,6 +12,6 @@ fi
 
 ./validate
 ./build
-./test
 ./package
+./test
```

Comment:
Why are we moving the test script to after the package script?

Reply:
This is so we use the `rancher-agent` image built by drone instead of `rancher-agent:latest`. This is so integration tests can written to test against latest changes of rancher-agent